src/pages/JobSearch/index.jsx

```diff
@@ -86,7 +98,9 @@ function JobSearch() {
           )}
         </Suspense>
       </Flex>
-      {/* @TODO Import Button component here once it's complete */}
+      <InnerBox>
```

Comment:
One more requirement: instead of disabling the button, can you hide the button when there are no job postings? This change will improve the user interface by removing unnecessary elements when they're not applicable. 

You can implement this using conditional rendering in React. Let me know if you have any questions about this implementation or want to discuss alternative approaches.